wasp/main.py

```diff
@@ -3,4 +3,5 @@
 
 import wasp
 from gadgetbridge import *
-wasp.system.schedule()
+
+wasp.system.run()
```

Comment:
Don't commit this sort of change! It can be useful for testing but isn't a good default.